__tests__/join/with-metadata/snapshot.txt

```diff
@@ -10,16 +10,19 @@ info:
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
-servers:
```

Comment:
Please add one more path item that doesn't have its own servers to make sure the root servers will be applied to it.

Reply:
done